MerkleTreeAirdrop/README.md

```diff
@@ -0,0 +1,49 @@
+# MerkleTreeAirdrop
+
+## Overview
+
+Requires a list of wallet address and root hash which are the leaves and merkle proofs needed to verify that a given data exists for that address in the tree.
+Contract acts as Airdrop and only sends out tokens on data verification i.e merkle proof and appropriate arguments used to build the merkle tree for that user.
```

Comment:
Please explain that it's an airdrop module for the ERC20 token.
And I think it would be better to include ERC20 in this module name.

Reply:
Good point, thanks for your review